src/js/actions/guides.js

```diff
@@ -313,6 +313,36 @@ define(function (require, exports) {
     setGuide.transfers = [resetGuidePolicies, deleteGuide];
 
     /**
+     * Clears all the guides in the given document
+     *
+     * @param {{id: number}} document Document model or an object containing document ID
```

Comment:
Furthermore, why not just `Document=` ?   It currently isn't even being used this way, right?  And in the "future" we'd probably have a full blown Document model.  I dunno.